packages/mongo-menu/src/MongoMenu.tsx

```diff
@@ -9,58 +9,61 @@ import {
   FocusableMenuItem,
 } from '@leafygreen-ui/menu';
 import { createDataProp } from '@leafygreen-ui/lib';
+import { useShowFocus } from '@leafygreen-ui/leafygreen-provider';
 import { uiColors } from '@leafygreen-ui/palette';
 import { css, cx } from '@leafygreen-ui/emotion';
 
 const iconRef = createDataProp('icon-ref');
 
-const buttonReset = css`
+const wrapperStyle = css`
   appearance: none;
   background: none;
   border: 0px;
   position: relative;
   padding: 0px;
 
+  &::-moz-focus-inner {
```

Comment:
Should this be handled in button now as well?

Reply:
Maybe once we convert button to use `useShowFocus`.